examples/basics/visuals/grid_mesh.py

```diff
@@ -0,0 +1,53 @@
+"""
+Demonstration of a mesh constructed in a grid about x,y,z coordinates.
+"""
```

Comment:
I could only find that the tests have a 'requires_scipy' decorator, and that one example in demo/gloo/unstructured_2d.py has this information in a comment at the top (but other examples don't seem to follow this pattern). Should I do something like the latter, or am I missing something?